### PR TITLE
[ID] Difficulties

### PR DESCRIPTION
**Pull request by fajar13k**
Add Bahasa Indonesia translation of `Difficulties` article

---

### CODE REVIEW

File: wiki/Difficulties/id.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- ![](/wiki/shared/diff/insane-m.png) Insane
+- ![](/wiki/shared/diff/expert-m.png) Expert
+- ![](/wiki/shared/diff/expertplus-m.png) Expert+
+<!-- markdownlint-enable MD045 -->
+
+## Star rating (tingkat kesulitan dalam skala bintang)
+
+![Jangkauan star rating](img/SR-range.png)
+
+On the website, beatmaps are automatically given a difficulty icon based on these star rating ranges:
```
Comment (HerendraTJ) [inline wiki/Difficulties/id.md]: Terjemahan sudah benar, tinggal bagian Inggris yang ini saja dihapus.

File: wiki/Difficulties/id.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+*Tidak sama dengan [Beatmaps](/wiki/Beatmaps).*
+
+**Kesulitan** beatmap menggambarkan tingkat keterampilan yang diperlukan untuk menyelesaikannya. Nama tiap tingkat dan *star rating* dari sebuah beatmap biasanya memberikan informasi yang cukup untuk mengetahui seberapa sulit sebuah beatmap.
+
+## Tingkat kesulitan
+
+*Lihat juga: [Penamaan Tingkat Kesulitan](/wiki/Ranking_Criteria/Difficulty_Naming)*
+
+Berikut adalah nama tingkat kesulitan yang biasa dimiliki dalam beatmap. Definisi dan persyaratan khusus dari masing - masing tingkat dapat ditemukan dalam [kriteria beatmap Ranked](/wiki/Ranking_Criteria) untuk masing - masing mode.
```
Comment (cl8n) [inline wiki/Difficulties/id.md]: just making sure is this "masing - masing" part correct? not sure what its function is in indonesian

File: wiki/Difficulties/id.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+Di situs web, beatmap secara otomatis diberikan ikon kesulitan berdasarkan rentang *star rating* ini:
+
+<!-- markdownlint-disable MD045 -->
+- ![](/wiki/shared/diff/easy-s.png) Easy: 0.0★–1.99★
+- ![](/wiki/shared/diff/normal-s.png) Normal: 2.0★–2.69★
+- ![](/wiki/shared/diff/hard-s.png) Hard: 2.7★–3.99★
+- ![](/wiki/shared/diff/insane-s.png) Insane: 4.0★–5.29★
+- ![](/wiki/shared/diff/expert-s.png) Expert: 5.3★–6.49★
+- ![](/wiki/shared/diff/expertplus-s.png) Expert+: 6.5★ and above
```
Comment (cl8n) [inline wiki/Difficulties/id.md]: translate "and above"

File: wiki/Difficulties/id.md (hunk 2 above)
Comment (fajar13k) [reply wiki/Difficulties/id.md]: It is correctly placed, each mode means masing - masing mode in Indonesian

Comment (HerendraTJ) [reply wiki/Difficulties/id.md]: each = masing masing

**Pull request merged**